src/index.js

```diff
@@ -0,0 +1,34 @@
+// import _ from 'lodash';
+
+import './style.css';
+
+const listTable = [{
+  description: 'go to the market',
+  completed: true,
+  index: 0,
+},
+{
+  description: 'wash dishes',
+  completed: true,
+  index: 1,
+},
+{
+  description: 'play guitar',
+  completed: true,
+  index: 2,
+},
+'rwanda',
```

Comment:
- [ ] Remove this, since it is returning undefined
![Screenshot from 2021-10-04 21-54-44](https://user-images.githubusercontent.com/62705947/135915821-4a9909fd-6e64-41cc-9448-6b58f626d2eb.png)
.